README.md

```diff
@@ -88,9 +88,24 @@ make html # create html documentation
 version-mp2-georchestra
 version-mp2
 ```
----
-**NOTE**
 
-When a stable branch is created, kindly update the `version.txt` with the new stable branch name and the same applies to Mapstore2 submodule
-
----
+## Release procedure
+
+All the changes are first delivered as release candidates, turned into release after user acceptance
+
+### Creating release candidate:
+- Once changes are ready to be delivered as release candidate, create new branch naming it as following `2022.02.xx-RC1-geOrchestra`,
+where `2022.02` corresponds to the major version of MapStore upstream; `RC1` is the number of release candidate.
+- Create new pre-release and tag for it at [Release creation page](https://github.com/georchestra/mapstore2-georchestra/releases/new),
+using following naming convention: `2022.02.00-RC1-geOrchestra`, where `2022.02` corresponds to the major
+version of MapStore upstream. `00` is the number of release for geOrchestra; `RC1` is the number of release candidate.
+- Add exhaustive description of the pre-release and attach all the binaries: *.war files for geOrchestra build and for corresponding MapStore build.
+
+### Creating release
```

Comment:
- Given a RC already available and accepted
- From the current stable branch corresponding to the series version to be released create new release and tag for it at [Release creation page](https://github.com/georchestra/mapstore2-georchestra/releases/new) using the following naming convention: `YYYY.MM.mm-geOrchestra`, where `YYYY.MM` corresponds to the major version and `mm` is the number of the minor release of the same series (eg. 2022.02.00-geOrchestra)
- Add exhaustive description of the release (including descriptions from all release candidates) and attach all the needed assets: 
   - *.war file for geOrchestra build
   - Link to the docker image in GeoSolutions Docker Hub
   - Link to the corresponding release page in MapStore as soon as available
   - Relevant configuration steps expected fro the new version